ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/admin/ApiPartyManagementService.scala

```diff
@@ -106,11 +64,19 @@ class ApiPartyManagementService private (
       maxWait: FiniteDuration,
       iteration: FiniteDuration => FiniteDuration): Future[AllocatePartyResponse] = {
     require(result.partyDetails.isDefined, "Party allocation response must have the party details")
-    pollUntilPersisted(result.partyDetails.get.party, minWait, maxWait, iteration).map {
-      numberOfAttempts =>
-        logger.debug(s"Party available, read after $numberOfAttempts attempt(s)")
+    val newParty = result.partyDetails.get.party
+    PollingUtils
+      .pollUntilPersisted(() => partyManagementService.listParties())(
```

Comment:
```suggestion
      .pollUntilPersisted(partyManagementService.listParties _)(
```
I guess it's a matter of taste, use your discretion.